mw-webapp/Dockerfile

```diff
@@ -1,8 +1,13 @@
 FROM node:20.11.0-alpine
-WORKDIR /mw-webapp
-COPY . .
+WORKDIR /masters-way
+COPY ./package.json .
+COPY ./pnpm-workspace.yaml .
+COPY ./pnpm-lock.yaml .
+COPY ./.eslintrc .
 RUN npm i -g pnpm@8.15.2
 RUN pnpm install --frozen-lockfile
+COPY ./mw-webapp ./mw-webapp
```

Comment:
it's strange a little that we need to copy global package.json for running webapp module. I think it is required because you removed dependencies from the webapp level. Let's move it back - they should be declared on the webapp level but installed in the rootby monorepo